claragenomics/dl4atac/dataset.py

```diff
@@ -121,12 +122,13 @@ def _get_generator(self):
                 sys.stdout.flush()
             rec = self.fh_data[file_id][local_idx - self.fh_indices[file_id][0]]
             sys.stdout.flush()
-            if len(rec.shape) == 1:
+            idx = yield {'idx':idx, 'x':rec}
+            #if len(rec.shape) == 1:
```

Comment:
can we delete the commented sections ?

Reply:
@tijyojwad do these commented lines serve any purpose?

Reply:
I deleted the commented lines. Have not deleted the DatasetEval section.

Reply:
Let's delete dataset eval.as well

Reply:
deleted.